static/templates/actors/character/partials/header.hbs

```diff
@@ -34,7 +34,7 @@
         </div>
         <div class="exp-data">
             <div class="exp-input">
-                <input name="system.details.xp.value" type="text" value="{{data.details.xp.value}}" data-dtype="Number" data-allow-delta placeholder="0" size="4" />
+                <input name="system.details.xp.value" type="text" value="{{data.details.xp.value}}" data-dtype="Number" data-allow-delta data-min="0" data-max="{{data.details.xp.max}}" placeholder="0" size="4" />
```

Comment:
I'm not 100% sure but there is probably a valid use case in not having a max, due to spillover.